v2x_ros_driver/README.md

```diff
@@ -0,0 +1,58 @@
+# v2x_ros_driver
+
+This package defines a carma-plaform compatible driver for Cohda OBUs though other models of OBU may work as well. The driver is compatible with DSRC and CV2X radios. The design documents for this driver can be found here <https://usdot-carma.atlassian.net/wiki/spaces/CRMMSG/pages/1319272562/Detailed+Design+-+DSRC+OBU+Driver>
+
+This driver is known to work successfully with the Cohda Wireless MK5 and MK6 OBU.  It implements the communication interface allowing UPER-encoded SAE J2735 communication to be exposed to a ROS network using [carma_driver_msgs/msg/ByteArray] (<https://github.com/usdot-fhwa-stol/carma-msgs/blob/develop/carma_driver_msgs/msg/ByteArray.msg>). It has been tested with SAE J2735 2016 but can support later versions as well.
+
+# Deployment Instructions
+
+The v2x-ros-driver is a ros2 package currently implemented in [ros2-foxy] (<https://docs.ros.org/en/foxy/Installation.html>). It creates a UDP client to listen to input from the v2x radio and broadcast the UPER encoded message to ROS.
+In order to deploy the driver in a ros2-foxy configured environment, the following steps can be used
+
+## Deploy using docker (recommended)
+
+1. Pull the latest docker image for driver from dockerhub
+
+```
+docker pull usdotfhwastol/v2x-ros-driver:latest
+```
+
+#### Note: The repository was recently renamed and until a new release is available, users may use usdotfhwastol/carma-cohda-dsrc-driver:latest which builds older versions of this code base but is manufacturer and DSRC/C-V2X agnostic.
```

Comment:
I receive the following error with this command:
```
docker pull usdotfhwastol/carma-cohda-dsrc-driver:latest
...
Error response from daemon: manifest for usdotfhwastol/carma-cohda-dsrc-driver:latest not found: manifest unknown: manifest unknown
```

Do you mean to point to the latest release version? i.e. usdotfhwastol/carma-cohda-dsrc-driver:<release-version>? Same comment applied to other instances of the `:latest` tag in this README.

Reply:
Yeah it seems like we do not have a latest tag for our release (which would be ideal), i will update this text to reference release tag from github

Reply:
Updated